astropy/units/physical.py

```diff
@@ -484,7 +485,7 @@ def get_physical_type(unit):
 
     Returns
     -------
-    physical_type : PhysicalType
+    physical_type : `PhysicalType`
```

Comment:
Absolute paths are better for xref.

```suggestion
    physical_type : `~astropy.units.PhysicalType`
```